mailpoet/assets/css/src/components/_top_bar.scss

```diff
@@ -84,14 +82,20 @@ $beamer-dot-size: 8px;
 .mailpoet-top-bar-beamer {
   align-items: center;
   background-color: $color-white;
-  border: solid 2px $color-tertiary-light;
-  border-radius: 4px;
+  border: none;
+  color: #787c82;
```

Comment:
Can we have a descriptive color variable instead?

Reply:
do you have a suggestion, or just `grey`? @samnajian

Reply:
Something like `$color-border-gray` or alike?